environment_ci.yml

```diff
@@ -9,6 +9,7 @@ dependencies:
   - coverage==4.0.3
   - python-coveralls==2.7.0
   - pytest==3.0.5
+  - flake8==3.2.1
```

Comment:
Pinned to this version as packages exists on Python 2.7, 3.4, 3.5, and 3.6 from `conda-forge`.